package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "ffc-demo-calculation-service",
   "description": "Digital service mock to claim public money in the event property subsides into mine shaft.",
-  "version": "4.3.7",
+  "version": "4.3.9",
```

Comment:
Hi @johnwatson484 , we had a patch 4.3.8 in the middle. Since there were changes to the application after that, the pipeline expected an incremented version. So had to bump it to 4.3.9.

Reply:
The pipeline expects a bump from what's in the main branch.  If this comparison is showing the main branch is 4.3.7 then should be ok for 4.3.8?

Reply:
Yeah we did that, but somehow the pipeline failed at Version Increment check stage. So had to do this as a work around.

Reply:
Changed the version back to v4.3.8 and it worked fine now. Thanks.